tribits/core/package_arch/TribitsAddExecutable.cmake

```diff
@@ -417,14 +417,10 @@ function(tribits_add_executable EXE_NAME)
 
   target_link_libraries(${EXE_BINARY_NAME} PUBLIC ${${PACKAGE_NAME}_LIBRARIES})
   foreach(depPkg IN LISTS ${PACKAGE_NAME}_LIB_ENABLED_DEPENDENCIES)
-    if (TARGET ${depPkg}::all_libs)
-      target_link_libraries(${EXE_BINARY_NAME} PUBLIC ${depPkg}::all_libs)
-    endif()
+    target_link_libraries(${EXE_BINARY_NAME} PUBLIC ${depPkg}::all_libs)
   endforeach()
   foreach(depPkg IN LISTS ${PACKAGE_NAME}_TEST_ENABLED_DEPENDENCIES)
-    if (TARGET ${depPkg}::all_libs)
-      target_link_libraries(${EXE_BINARY_NAME} PUBLIC ${depPkg}::all_libs)
-    endif()
+    target_link_libraries(${EXE_BINARY_NAME} PUBLIC ${depPkg}::all_libs)
```

Comment:
Redundant loops - can be combined into a single `foreach(IN LISTS)` loop.

Reply:
Resolved in commit https://github.com/TriBITSPub/TriBITS/pull/479/commits/21d3c4b32e4e09a6ad1cd82205f36fe1f14dbca7.  Thanks for the suggestion!